mne/utils/check.py

```diff
@@ -447,3 +447,12 @@ def _check_option(parameter, value, allowed_values):
         options += ' and %r' % allowed_values[-1]
     raise ValueError(msg.format(parameter=parameter, options=options,
                                 value=value))
+
+
+def _check_all_same_channel_names(instances):
+    """Check if a collection of instances all have the same channels."""
+    ch_names = list(instances)[0].info["ch_names"]
```

Comment:
Why do you need to make it a list?

Reply:
I wanted to cover the dict case too.

Reply:
Actually it doesn't solve that case ... this would return the keys. I guess both `list` calls are dumb. Will remove.